homeassistant/components/webostv/media_player.py

```diff
@@ -219,11 +222,21 @@ async def async_update(self):
             ):
                 pass
 
+    @property
+    def unique_id(self):
+        """Return the unique id of the device."""
+        return self._name
+
     @property
     def name(self):
         """Return the name of the device."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the device."""
+        return "hass:television"
```

Comment:
Set the media player device class instead.

https://developers.home-assistant.io/docs/entity_media_player#available-device-classes